scripts/main/view.js

```diff
@@ -1629,3 +1632,53 @@ view.diagnostics = {
 	},
 
 };
+
+view.update = {
+	init: function () {
+
+		multiselect.clearSelection();
+
+		view.update.title();
+		view.update.content.init()
+
+	},
+
+	title: function () {
+
+		lychee.setTitle('Update', false)
+
+	},
+
+	clearContent: function () {
+		let html = '';
+		html += '<pre class="logs_diagnostics_view"></pre>';
+		lychee.content.html(html);
+	},
+
+	content: {
+		init: function () {
+			view.update.clearContent();
+
+			// code duplicate
+			api.post('Update::Apply', [], function (data) {
+				html = view.preify(data, 'logs_diagnostics_view');
+				lychee.content.html(html);
+			});
+		}
+	},
+};
+
+view.preify = function (data, css) {
+	html = '<pre class="' + css + '">';
+	if (Array.isArray(data)) {
+		for (let i = 0; i < data.length; i++) {
+			html += '    ' + data[i] + '\n';
+		}
+	} else {
+		html += '    ' + data_json;
```

Comment:
Isn't `data_json` going to be undefined here?

Reply:
oups ^^